deployments/server/templates/deployment.yaml

```diff
@@ -55,8 +55,18 @@ spec:
           containerPort: {{ .Values.adminPort }}
           protocol: TCP
         - name: km-metrics
-          containerPort: {{ .Values.kubernetesManagerMetricsPort }}
+          containerPort: {{ include "inference-manager-server.metricsPort" . }}
```

Comment:
I might be missing, but this is not defined?

Reply:
It's defined in the helper template.

Reply:
ah i see. nice!